spec/features/audit/report/progress_spec.rb

```diff
@@ -1,72 +1,102 @@
 RSpec.feature "Reporting on audit progress" do
-  let!(:my_organisation) do
-    create(
-      :organisation
-    )
+  scenario "Displaying the number of items included in the audit" do
+    given_i_am_an_auditor_belonging_to_an_organisation_with_three_content_items
```

Comment:
Good change. Could probably be more specific still? Sorry, I should have pointed this out in my last review. You could do something like:

> Given I ... three transactions
> ...
> When I filter by Transaction document type
> ...
> When I filter by Guide document type
> ...

Reply:
Have added an additional step in which I update the content items to give them `document_type: 'transaction'`. I just did this to give the test an extra step to improve clarity without taking away from the re-usability of the `given...` method. What do you think?